addOns/formhandler/src/main/java/org/zaproxy/zap/extension/formhandler/FormHandlerParam.java

```diff
@@ -35,6 +37,8 @@
 public class FormHandlerParam extends VersionedAbstractParam {
 
     private static final Logger LOGGER = LogManager.getLogger(FormHandlerParam.class);
+    private static final Faker FAKER = new Faker(new Random(1));
```

Comment:
I wasn't sure how static/consistent we wanted to be with the generated values. This ensure they're consistent across all installs (might be overkill).

Reply:
I thought this was going to be used in an option in which the user could generate values for the fields, not to add defaults. (Defaults should have fixed values in any case.)

Reply:
I guess I wasn’t specific enough about my plan.

I guess I’m this case I could just generate the values once and provide them literally if we want them to be that consistent.

I’ll put more thought into leveraging it in a user driven way, for another PR.

Reply:
Sounds good. I think it's better if the defaults don't change between runs (less chances of breaking randomly).

Reply:
Well even if we allow randomness it would just be when initially set. But they'd potentially be different between your machine, mine, psiinon, whoevers.

Reply:
But in CI that would be always, it would use a new home each time.

Reply:
Good point.